inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.h

```diff
@@ -97,28 +97,29 @@ class MKLDNNEltwiseNode : public MKLDNNNode {
     static bool isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept;
 
 private:
-    struct EltwisePrim {
-        EltwisePrim(size_t batch) : batchDimIdx(batch) {}
+    struct EltwiseExecutor {
+        EltwiseExecutor(size_t batch) : batchDimIdx(batch) {}
         virtual void exec(const MKLDNNEltwiseNode& node, const jit_eltwise_call_args_ptrs &args_ptrs, const VectorDims &dims_out) = 0;
         virtual const jit_eltwise_params& getJep() const = 0;
-        virtual ~EltwisePrim() = default;
+        virtual ~EltwiseExecutor() = default;
 
         size_t batchDimIdx = 0;
     };
-    using primPtr = std::shared_ptr<EltwisePrim>;
+    using primPtr = std::shared_ptr<EltwiseExecutor>;
```

Comment:
We should rename `primPtr`, `pPrim` since we do not use "primitive" term in this context.